.env.example

```
@@ -0,0 +1,3 @@
+# Sentry DSN — get this from sentry.io project settings
+# Leave empty to disable Sentry (safe for local development)
+VITE_SENTRY_DSN=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)